### DISCUSSION TITLE
Slow

### DISCUSSION THREAD

**Discussion by ChesDonno** [Chia-Network/chia-blockchain#3996] [Q&A] [answered]
Perhaps I'm doing something wrong here.

SPEC:
Ubuntu 20.04
Ryzen 9 3900 12c/24t
3200Mhz 14CL 32G (4x8g)
2T inland NVMe
8T Seagate HDD

chia software 1.1.4 

I started parallel plotting at May 5th 2021 19:15
now(May 6th 16:00) it at 31%

Plot settings:
Plot = 6
RAM = 4g
Thread = 6

This is not 8 to 10 hours, this is almost 24 hrs. Is that normal?

**Comment by moshenskyDV** [comment, marked as answer]
If I get it correctly you are running 6 plots in parallel?
If so - it's to greedy. 2Tb SSD can run just 5 plots in parallel (probably you missed with math. Check the difference between gigabytes and gibibytes).

**Reply by ChesDonno** [reply]
I'll double check this again.
